releasenotes/notes/asm-1-click-activation-with-RCM-6a2431949768b73b.yaml

```diff
@@ -1,4 +1,6 @@
 ---
 features:
   - |
-    ASM: 1-click activation with RCM.
+    ASM: 1-click activation ability through Remote Configuration Management. 
```

Comment:
I don't believe we have an explanation of ASM in this release note or any previous one.